libraroo-backend/src/physical-books/physical-books.controller.ts

```diff
@@ -0,0 +1,4 @@
+import { Controller } from '@nestjs/common';
+
+@Controller('physical-books')
+export class PhysicalBooksController {}
```

Comment:
![image](https://github.com/user-attachments/assets/1ec2d1d0-55bc-4c6d-9be6-60567159c981)

Reply:
We didnt discuss specifically what we need